graphql/schema.graphql

```diff
@@ -49,6 +49,13 @@ type Patch {
   variantsTasks: [VariantTask]!
   activated: Boolean!
   alias: String!
+  time: PatchTime
```

Comment:
These fields are duplicates of fields already in the schema. I also don't see the appeal in delivering them formatted - yes you'll be able to skip formatting on the client, but that just makes it less flexible for other use cases and doesn't reduce the total amount of work

Reply:
they're not duplicates, they are the fields that no longer map to the model and are generated from a resolver. The appeal in delivering them formatted is very visible to me. 
1. don't have to use moment on client. huge win. 
2. they're already formatted

How does it make it less flexible for other use cases? what are the other use cases? if in the future the un-formatted dates are needed on client for some reason then they can be added to the patch schema. that's the magic about gql. we can include fields, we can not include fields. it's flexible.

Reply:
removed fields `createTime`, `startTime` and `finishTime` from the schema